pkg/clients/export/distro/client.go

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
```

Comment:
If that's the case, should this be a service-to-service call at all? Because it's effectively encapsulated within export-client, making it a responsibility of the export-client service. Also, in the previous implementation, the host/port were initialized once and would never be updated. This way, the endpoint monitor will keep the endpoint up to date. If this change isn't made then we have only one place in the whole platform where a service to service call is made that doesn't follow the client pattern which, to me, says it needs to be refactored some other way.

Reply:
I always thought that export-client and export-distro should be merged, that it doesn't makes any sense to have these two services separated. 
This call is used to notify export-distro that the export configuration has been changed, so only export client will use this api. So I prefer to not export the export-distro api-client to third party code, as it should not be used by them.

Anyway, you can merge this code and I will try to move it later to make it private to export.